chalice/deploy/swagger.py

```diff
@@ -128,6 +129,11 @@ def _generate_route_method(self, view):
             'x-amazon-apigateway-integration': self._generate_apig_integ(
                 view),
         }  # type: Dict[str, Any]
+        if view.view_function.__doc__:
+            doc_lines = inspect.getdoc(view.view_function).splitlines()
+            current['summary'] = doc_lines[0]
+            if len(doc_lines) > 1:
```

Comment:
`if doc_lines:` is more succinct.

Reply:
but I'm looking for > 1, not > 0 right?

Reply:
I'm distinguishing single line docstrings from multiline.

Reply:
My mistake.

Reply:
np ;-)